specs/protocol/deposits.md

```diff
@@ -304,6 +304,8 @@ The predeploy stores the following values:
   - `overhead` (`uint256`): The L1 fee overhead to apply to L1 cost computation of transactions in this L2 block.
   - `scalar` (`uint256`): The L1 fee scalar to apply to L1 cost computation of transactions in this L2 block.
 - With the Ecotone upgrade, the predeploy additionally stores:
```

Comment:
Ah good catch, we should be really deleting line 306

Reply:
We are breaking up upgrade specific things per hardfork, there is a `ecotone` directory now

Reply:
> We are breaking up upgrade specific things per hardfork, there is a `ecotone` directory now

Sensible. There are lots of places in the spec that currently have "oh but with ecotone...".

I'll just leave this here and you should feel free to close or modify it to your purposes.